server.py

```diff
@@ -0,0 +1,11 @@
+from flask import Flask, request
+from main import main
+
+app = Flask(__name__)
+
+@app.route('/', methods = ['POST'])
+def process_image():
+    return main(request.get_json()["data"])
```

Comment:
@guinnesschen `main.main()` not a helpful name, especially when it's not the "main" entry-point of the API